inc/block-editor-panels/register-meta.php

```
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Register post meta for REST API access.
+ *
+ * @package vk-all-in-one-expansion-unit
+ */
+function veu_register_block_editor_meta() {
+	$post_types = get_post_types( array( 'public' => true ) );
```

PHPStan (2.1.46)

[error] Function get_post_types not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)